README.md

```diff
@@ -34,18 +34,14 @@ change.
 
 ### Conventions
 
-Each entry is of the form:
-```
-* [The Literal Title of The Book](https://a-link-to-the/book/) - an optional comment if the title isn't self-explanatory
-```
-
-The `.editorconfig` file captures basic source files formatting conventions.
-Many editors support this file natively; others (such as VS code) require a plugin, see https://editorconfig.org/.
-
-The default branch is `main`. See [Renaming the default branch from master](https://github.com/github/renaming) for the rationale.
+Basic conventions around source files formatting are captured in the
```

Comment:
@kw-fn Why did you delete the entry format convention? I think PR authors need to know the intended format so they can adhere to it. Otherwise they have to guess. Please could you put that back in?

Reply:
I put it back in.

Reply:
There is only a single branch to the project as far as I can determine. Also I believe the use of a plugin should not necessarily be a recommendation given that simple editing oversight can accomplish what the dot-file seeks to enforce.

Reply:
Yes, there's a single branch, but the adoption of `main` as the default branch is a relatively recent convention which I wanted to explain.

The editor config file is a general hygiene item. It's not that important, but I'd like to keep it because I want to promote its use in other repos.